tensorflow/contrib/tensor_forest/kernels/stats_ops.cc

```diff
@@ -305,11 +303,11 @@ class ProcessInputOp : public OpKernel {
         leaf_examples[leaf_ids(i)].push_back(i);
       }
       for (const auto& leaf : leaf_examples) {
+        const int leaf_id = leaf.first;
         // choose a feature to be used for each split.
         for (int i = 0; i < num_splits_to_consider; ++i) {
           int rand_feature = rng_->Uniform(num_total_features);
-          const int leaf_id = leaf.first;
-          leaf_feature_to_use_per_split[leaf_id].push_back(rand_feature);
+          features_per_split[leaf_id].push_back(rand_feature);
         }
```

Comment:
u can probably also get a ref to features_per_split[leaf_id[ in line 306 and use that here...